Docs: Add rspack community collab (#80217)
vercel/next.js: docs/04-community/02-rspack.mdx

---
title: Rspack Integration
nav_title: Rspack

> description: Use the `next-rspack` plugin to bundle your Next.js with Rspack.

Reviewer: You can add in experimental here as the page type (e.g. similar to canary)

---

The Rspack team has created a community plugin for Next.js, which is part of a [partnering effort](https://rspack.rs/blog/rspack-next-partner) with the Rspack team.

> This plugin is currently experimental. Please use this [discussion thread](https://github.com/vercel/next.js/discussions/77800), to give feedback and triage any issues you might find.

Reviewer: ```suggestion
This plugin is currently experimental. Please use this [discussion thread](https://github.com/vercel/next.js/discussions/77800) to give feedback on any issues you might find.
```

> Read the [Rspack docs](https://rspack.rs/guide/tech/next), and try out [this example](https://github.com/vercel/next.js/tree/canary/examples/with-rspack).

Reviewer: ```suggestion
Learn more on the [Rspack docs](https://rspack.rs/guide/tech/next) and try out [this example](https://github.com/vercel/next.js/tree/canary/examples/with-rspack).
```

---
title: Rspack Integration
nav_title: Rspack
description: Use the `next-rspack` plugin to bundle your Next.js with Rspack.
version: experimental
---

The Rspack team has created a community plugin for Next.js, which is part of a [partnering effort](https://rspack.rs/blog/rspack-next-partner) with the Rspack team.

> This plugin is currently experimental. Please use this [discussion thread](https://github.com/vercel/next.js/discussions/77800) to give feedback on any issues you might find.

Reviewer 2: ```suggestion
This plugin is currently experimental. Please use this [discussion thread](https://github.com/vercel/next.js/discussions/77800) to give feedback on any issues you encounter.
```

---
title: Rspack Integration
nav_title: Rspack

> description: Using the next-rspack plugin to use Rspack as bundler

Reviewer 3: Should next-rspack be inline code

Reviewer 3: Maybe to avoid "using" twice
Using the next-rspack plugin to enable Rspack as bundler

---

> The Rspack team has created a community plugin for Next.js.

Reviewer 3: Maybe tag this along
The Rspack team has created a community plugin for Next.js, which is part of a [partnering effort](https://rspack.rs/blog/rspack-next-partner) with the Rspack team
